startup/80-areadetector.py

```diff
@@ -18,6 +18,8 @@
 from ophyd import StatusBase
 from ophyd.status import DeviceStatus
 from nslsii.ad33 import SingleTriggerV33, StatsPluginV33
+
+from packaging.version import Version
 from distutils.version import LooseVersion
```

Comment:
This import from `distutils` may start failing at some point. Probably worth removing if it's not used.